09Ianuarie/src/ro/sci/ale/ian09/max/MaxNumberTest.java

```diff
@@ -0,0 +1,34 @@
+package ro.sci.ale.ian09.max;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static ro.sci.ale.ian09.max.MaxNumber.getMax;
+
+class MaxNumberTest {
+
+    @org.junit.jupiter.api.Test
+    void main() {
+        int num1 = 10;
+        int num2 = 5;
+        int num3 = 8;
+        int result = getMax(getMax(num1, num2), num3);
+
+        assertEquals(10, result);
+    }
+
+    @Test
+    void mainEquals() {
+        assertEquals(10, getMax(getMax(10, 10), 10));
+    }
```

Comment:
you are testing the getMax which has 2 parameters. No need to test with 3 numbers.